test/uk/gov/hmrc/merchandiseinbaggage/stubs/MibBackendStub.scala

```diff
@@ -141,7 +142,7 @@ object MibBackendStub extends MibConfiguration with CoreTestData {
   def givenEoriIsChecked(eoriNumber: String)(implicit server: WireMockServer): StubMapping =
     server
       .stubFor(
-        get(urlPathEqualTo(s"$checkEoriUrl$eoriNumber"))
+        get(urlPathEqualTo(s"${config.mibCheckEoriUrl}$eoriNumber"))
```

Comment:
This can just be `get(urlPathEqualTo(s"/declare-commercial-goods/validate/eori/$eoriNumber"))`